config/postman/collection.json

```diff
@@ -178,7 +178,33 @@
         }
       },
       "response": []
+    },
+    {
+      "name": "UpdateProfile",
+      "request": {
+        "method": "PATCH",
+        "header": [],
+        "body": {
+          "mode": "raw",
+          "raw": "{\r\n    \"name\": \"MyName\",\r\n    \"lastname\": \"MyLastname\",\r\n    \"gender\": \"MyGender\",\r\n    \"birthDate\": \"MyBirthDate\"\r\n}",
```

Comment:
"birthDate": "MyBirthDate"  // is not a valid date value, replace it with a valid date